### DISCUSSION TITLE
Cost of API calls

### DISCUSSION THREAD

**Discussion by yangpuhai** [BAAI-Agents/Cradle#7] [Q&A] [answered]
Looks like an expensive work. 
How much did you spend on API calls? 
To be more specific, how much does it cost to run the game for ten minutes?
Thanks!

**Comment by tellarin** [comment, marked as answer]
Hej @yangpuhai,

it can indeed get costly calling the backend model many times.

We can't offer a precise estimate just based on time. Many factors will affect costs. For example: your service tier and limits at the model provider, latency, if you're using concurrent calls or not, and even what part of the game is being played.
